addons/website_livechat/models/mail_channel.py

```diff
@@ -71,6 +71,7 @@ def close_livechat_request_session(self, type='leave', **kwargs):
             self.message_post(author_id=self.env.ref('base.user_root').sudo().partner_id.id,
                               body=leave_message, message_type='comment', subtype='mt_comment')
 
+    @api.returns('mail.message', lambda value: value.id)
```

Comment:
It's strange because the docstring of `returns` states that override are supposed to inherit from it.

It's worth notifying the framework python team of the potential issue. Either the docstring is obsolete or the feature is broken.

Reply:
As confirmed by them, the "inherit" means "python inherit" in this case and not "model inherit", which validates the need for our fix.